react/src/lib/Menu/index.js

```diff
@@ -251,7 +251,8 @@ class Menu extends React.Component {
         flag = true;
         break;
 
-      case 37: //left
+      case 27:
+      case 37: //escape or left
```

Comment:
The handling in this case checks for a submenu and then closes it. Currently pressing left key closes the submenu and escape is expected to do the same.